documents/beta/custom_race_logos.yaml

```diff
@@ -0,0 +1,68 @@
+# Original author: Magnemania
+
+# This creates one campaign for each race that takes the shape of the respective race's logo
+# As with all custom mission orders, this requires setting "mission_order: custom" in your YAML to work.
+# To play with less than all of the races, remove that race's respective campaign from the custom mission order.
+# To have enough missions of each faction, it's also necessary to have race-swaps enabled.
+
+# Mission unlocks follow a grid pattern, with the ability to jump over gaps in orthogonal directions up to 3 slots away.
+
+# Contains:
+# terran missions: 29
+# protoss missions: 30
+# zerg missions: 24
```

Comment:
these counts seem wrong; I count 31 zerg and 24 protoss. Could also just remove these lines.